### DISCUSSION TITLE
Multithread makes lazer stutter a lot

### DISCUSSION THREAD

**Discussion by kaikene** [ppy/osu#25619] [Q&A]
I used to play with multithread like normal and nothing used to happend, but a few releases ago something happend and now i cant play with it enabled for some reason. The solution is to just play with Singlethread obviously but i still want to ask if i have to do something specific to fix it or is this an actual problem with osu itself, i didnt want to open an issue since this seems very specific to me and the problem is probably not reproducible. I will leave the logs from playing with Multithread enabled and two videos showing the difference.

Logs:
[network.log](https://github.com/ppy/osu/files/13516143/network.log)
[runtime.log](https://github.com/ppy/osu/files/13516145/runtime.log)
[performance.log](https://github.com/ppy/osu/files/13516146/performance.log)
[database.log](https://github.com/ppy/osu/files/13516147/database.log)
[updater.log](https://github.com/ppy/osu/files/13516148/updater.log)

Video (Multithread):
https://github.com/ppy/osu/assets/35354910/74478c8d-998b-4699-b92c-82d8e9312ef6

Video (Singlethread):
https://github.com/ppy/osu/assets/35354910/bd5e9a28-9d0b-4ae7-b2ef-2a4c20a05545

Note: Since this is recorded the FPS is obviously lower than normal, but it kinda helps to show that multithread does have bad performance on my pc now, single thread has "normal" fps while multithread stutters like crazy

**Comment by bdach** [comment]
The videos attached show that the input thread is dying of death.

Did you change something in hardware recently? Get a new mouse/tablet? Install new drivers? Change polling rate of something to 1000Hz or above?
